.circleci/config.yml

```diff
@@ -92,32 +101,21 @@ commands:
             node --version
             pnpm --version
 
-      - when:
-          condition:
-            equal: [stable, << parameters.react-version >>]
-          steps:
-            - run:
-                name: Install js dependencies
-                command: pnpm install
-      - unless:
-          condition:
-            equal: [stable, << parameters.react-version >>]
-          steps:
-            - run:
-                name: Install js dependencies without frozen lockfile
-                command: pnpm install --no-frozen-lockfile
       - run:
-          name: Resolve React version
+          name: Install js dependencies
           command: |
-            pnpm use-react-version
-            # log a patch for maintainers who want to check out this change
-            git --no-pager diff HEAD
-      - when:
-          condition: << pipeline.parameters.with-material-ui-6 >>
-          steps:
-            - run:
-                name: Install @mui/material v6
-                command: pnpm use-material-ui-v6
+            args=""
+            if [ -n "$REACT_VERSION" ] && [ "$REACT_VERSION" != "stable" ]; then
+              args="react@$REACT_VERSION"
+            fi
+            if [ -n "$MATERIAL_UI_VERSION" ] && [ "$MATERIAL_UI_VERSION" != "stable" ]; then
+              args="$args @mui/material@$MATERIAL_UI_VERSION @mui/system@$MATERIAL_UI_VERSION @mui/icons-material@$MATERIAL_UI_VERSION @mui/utils@$MATERIAL_UI_VERSION @mui/material-nextjs@$MATERIAL_UI_VERSION @mui/lab@latest"
```

Comment:
Does it make sense to always use `latest` for `@mui/lab`?

Reply:
I don't know, I just copied what was already there. It was introduced in https://github.com/mui/mui-x/pull/15342

edit: I think it's supposed to be `latest-v6`